src/client.rs

```diff
@@ -36,57 +38,67 @@ impl fmt::Debug for Client {
     }
 }
 
-impl Default for Client {
+impl Default for Client<Token> {
```

Comment:
This should probably be replaced by `impl<T: Default> for Client<T>`, so that it works for user provided `RefreshableToken`s as well.